docs/data/Allas/using_allas/rclone.md

```diff
@@ -49,23 +49,23 @@ The default protocol of Allas is Swift. In Puhti and Mahti Swift based Allas con
 module load allas
 allas-conf
 ```
-The `allas-conf` command asks for your CSC password (the same you use to login to CSC servers). It lists
+The `allas-conf` command asks for your CSC password (University/Haka password will not work here). It lists
 your projects in Allas and asks you to define the project that will be used. Then _allas-conf_ generates a Rclone configuration file for the Allas service and authenticates the connection to the selected project. In Rclone command this swift based connection is referred with remote name `allas:`. The authentication information is stored in the shell variables `OS_AUTH_TOKEN` and `OS_STORAGE_URL` that are valid for up to eight hours. However, you can refresh the authentication at any time by running _allas-conf_ again. The environment variables are available only for that login session, so if you login to Puhti in another session, you need to authenticate again to access Allas.
 
 ### Rclone with S3 on CSC supercomputers
 
 If you want to use Allas with S3 protocol in stead run the allas-conf command with option --mode s3cmd.
```

Comment:
```suggestion
If you want to use Allas with the S3 protocol instead, run the allas-conf command with the `--mode S3` option.
```

Reply:
Since the example below was changed from `s3cmd` to `S3`.

Reply:
Good